ENVIRONMENT.md

```diff
@@ -72,6 +72,8 @@ This requires:
 - `HS_EMAIL`: HDX Signals email used as the sender in any Mailchimp campaigns.
 - `HS_SURVEY_LINK`: Link to the Mailchimp feedback survey contained at the bottom of
 all emails.
+- `HS_ADMIN_NAME`: Name to be recorded for user who triaged and sent email. It
+does not get recorded unless the analyst goes through the triaging process.
```

Comment:
Sorry for being picky, I think (not sure) that the code does not run if HS_ADMIN_NAME is not defined and in that case I would write: 
"Name to be recorded for user who triaged and sent email. It's required to complete the triaging process."

Reply:
i like the suggestion - implemented [66a483c](https://github.com/OCHA-DAP/hdx-signals/pull/283/commits/66a483c203c4ef4a1d2d02cd78dc8caea60432dc)